docs/index.md

```diff
@@ -5,60 +5,49 @@ page_keywords: documentation, docs,  docker, compose, orchestration, containers
 
 ## Overview
 
-Compose is a tool that allows you to orchestrate multiple Docker containers.
-With Compose, you can build clusters of containers which provide the resources
-(services, volumes, etc.) needed to build and run a complete distributed
-application.
+Compose is a tool for defining and running complex applications with Docker. It
+lets you define a multi-container application in a single file, then spin it up
+in a single command which does everything that needs to be done to get it
```

Comment:
Ditto, "it"-->"the app"

Also, can simplify "needs to be done" to "needed", so "...a single command that does everything needed to get the app runing."